lib/scripts/detect_buildpack.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022-Present Datadog, Inc.
+BUILD_DIR="${BUILD_DIR:-/tmp/app}"
```

Comment:
```suggestion
VCAP_DIR="/home/vcap"
```
The `staging_info.yml` file is always extracted to the `/home/vcap` directory.

See https://docs.cloudfoundry.org/buildpacks/understand-buildpacks.html#-droplet-filesystem